net-mgmt/net-snmp/src/opnsense/service/templates/OPNsense/Netsnmp/snmpd.conf

```diff
@@ -1,8 +1,23 @@
 {% if helpers.exists('OPNsense.netsnmp.general.enabled') and OPNsense.netsnmp.general.enabled == '1' %}
 
-{% from 'OPNsense/Macros/interface.macro' import physical_interface %}
-
+{% if helpers.exists('OPNsense.netsnmp.general.listen') and OPNsense.netsnmp.general.listen != '' %}
+{% if helpers.exists('OPNsense.netsnmp.general.listen') %}
```

Comment:
why check twice?

Reply:
One for v4 and other for v6

Reply:
sure, but you check helpers.exists('OPNsense.netsnmp.general.listen')  three times -- it's only required once

Reply:
better?